pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/errorprone/xml/InvalidSlf4jMessageFormat.xml

```diff
@@ -375,6 +375,383 @@ public final class Main {
                 public static void main(String[] args) {
                     final Logger logger = LoggerFactory.getLogger(LoggerHelper.class);
 
+                    logger.info(pattern, 1, 2);
+                }
+            }
+        ]]></code>
+    </test-code>
+    <test-code>
+        <description><![CDATA[
+missing argument
```

Comment:
you seem to be repeating the descriptions from the original SLF4J scenarios, make sure to flag all these as `log4j2`, so upon a failure a developer can quickly detect which one is having trouble